lib/Itho/CC1101Packet.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Author: Klusjesman, modified bij supersjimmie for Arduino/ESP8266
+ */
+
+#ifndef CC1101PACKET_H_
+#define CC1101PACKET_H_
+
+#include <stdio.h>
+#include <Arduino.h>
+
+#define CC1101_BUFFER_LEN        64
+#define CC1101_DATA_LEN          CC1101_BUFFER_LEN - 3
+
+
+class CC1101Packet
+{
+	public:
+		uint8_t length;
+		uint8_t data[128];
```

Comment:
Why is this packet not initialized?

```c++
uint8_t length = 0;
uint8_t data[128] = {0};
```

Reply:
It should and looking through the code it indeed isn't. Probably something that has been in there from the first version of the lib (multiple devs ago... ;-))